boards.txt

```diff
@@ -24623,6 +24623,230 @@ nebulas3.menu.EraseFlash.all=Enabled
 nebulas3.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+<<<<<<< HEAD
```

Comment:
Remove this. It's some leftover from the conflicts.
```suggestion
```

Reply:
resolved